lib/puppet/defaults.rb

```diff
@@ -742,10 +742,10 @@ def self.default_basemodulepath
 
         Defaults to the node's fully qualified domain name.",
       :hook => proc { |value| raise(ArgumentError, _("Certificate names must be lower case")) unless value == value.downcase }},
-    :dns_alt_names => {
-      :default => '',
-      :desc    => <<EOT,
-A comma-separated list of alternate DNS names for Puppet Server. These are extra
+    :subject_alt_names => {
+        :default => '',
+        :desc => <<EOT,
+A comma-separated list of alternate names for Puppet Server. These are extra
```

Comment:
If this is now for `subject_alt_names` we should update this to mention that it can be `IP` or `DNS` type when prefixed with the type and a colon, with un-typed inputs defaulting to the `DNS` type.